src/biogeophys/SnowHydrologyMod.F90

```diff
@@ -800,12 +808,12 @@ subroutine BuildFilter_SnowpackInitialized(bounds, num_c, filter_c, &
           ! maintain answers as before.
           snowpack_initialized(c) = ( &
                snl(c) == 0 .and. &
```

Comment:
@billsacks I think the next four 0.01 values are what you wanted me to look for, right?

More generally speaking, I think this code is ready for review. Also I would like to know your preference in this case between new Unit Test and new standard test-suite test. I will be working on that next.

Reply:
I looked for all the instances of 0.01 at the time, and there were some more that looked unrelated, so I left them alone.